orders/tests.py

```diff
@@ -0,0 +1,62 @@
+import datetime
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+from .models import Order
+from .models import StatusOptions
+import pytest
+
+NUMBER_OF_DAYS = 3
```

Comment:
since those should be used only on `create_order`, I see no need for them to be constants

Reply:
> since those should be used only on `create_order`, I see no need for them to be constants

There's a test that calculate the total price of the order, and it's using this constant as well